hack/lib/golang.sh

```diff
@@ -207,6 +207,9 @@ sedna::golang::build_binaries() {
   goldflags="${GOLDFLAGS=-s -w -buildid=} $(sedna::version::ldflags)"
   gogcflags="${GOGCFLAGS:-}"
 
+  CGO_ENABLED=0
+  GOARCH=arm64
```

Comment:
Only build arm64, how about other archs, e.g. `x86`?

Reply:
delete it